pkg/manager/member/tiflash_member_manager.go

```diff
@@ -281,6 +281,12 @@ func getNewHeadlessService(tc *v1alpha1.TidbCluster) *corev1.Service {
 					TargetPort: intstr.FromInt(int(20170)),
 					Protocol:   corev1.ProtocolTCP,
 				},
+				{
```

Comment:
There are two metrics ports for TiFlash metrics, refer to https://github.com/pingcap/tidb-operator/blob/master/pkg/manager/member/tiflash_member_manager.go#L448

Reply:
done